### DISCUSSION TITLE
Sponsoring?

### DISCUSSION THREAD

**Discussion by smileBeda** [bootscore/bootscore#584] [Ideas]
Hello folks of Bootscore.
I can see you setup something nice here - and mostly, free. I am a developer and perfectly aware of how much work it is even to maintain a plugin, but you maintain a theme _and_ several plugins.

I wanted to ask, if you have other sources of sponsoring other than the 2 pro plugins, if you foresee issues in future financially that might force you to abandon the project, and if you have some ways to be sponsored. I own two companies and always am ready to sponsor great projects like this one. For example we sponsor [PHPDocumentor](https://github.com/phpDocumentor), but also other, non-WP related projects.

Thus my question basically is, if you have some means to get sponsored, and if it is even something you consider.

Thank you! Also for the great software, it seems we found our new base theme :)

**Comment by smileBeda** [comment]
yeah.. I mean, sometimes I wonder if my glasses are made of tinfoil or actual corrected glass. Just saw the sponsoring button on the main GitHub Page. 🤷 

Thanks and sorry - can be closed.

**Comment by crftwrk** [comment]
No, you're absolutely right, your glasses are fine ;-)

There was no sponsor button before. We had this on our todo list https://github.com/bootscore/bootscore/issues/187 since a while but failed to setup sponsor feature. This discussion was the latest reminder that we should enable sponsorship, because other users asked as well. So, finally we finished setup yesterday.

Tank you for this reminder!
